themes/vocabulary_theme/templates/layout.html

```diff
@@ -138,19 +156,89 @@ <h1 class="page-title pb-3 mb-4">{% block header %}{{ this.title }}{% endblock %
       </div>
     </div>
   </div>
-  <footer class="main-footer bg-dark">
-    <div class="container-fluid">
-      <div class="row justify-content-md-center">
-        <div class="col-lg-9 col-md-9 col-sm-12 footer text-light py-4 px-3">
-          <small>
-            <p><a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License"
-                  style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a></p>
-            <p class="text-muted">All the content on this website is licensed under a <strong><a rel="license"
-                  href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International
-                  License</a></strong> unless otherwise specified.</p>
-          </small>
+  <footer class="main-footer">
+    <div class="container">
+      <div class="columns">
+        <div class="column">
+          <a href="https://creativecommons.org" class="main-logo margin-bottom-bigger">
+            <svg xmlns="http://www.w3.org/2000/svg" preserveAspectRatio="xMidYMid meet" viewBox="0 0 304 73">
+              <use href="/cc.svg#logomark"></use>
```

Comment:
See the [Vocabulary Asset Usage guide](https://cc-vocabulary.netlify.app/?path=/docs/vocabulary-usage--page#assets-1).